client/src/components/AvatarCanvas/index.tsx

```diff
@@ -37,41 +39,26 @@ function AvatarCanvas({
 
   return (
     <>
-      <Draggable position={selfPositionRef.current} onPositionChange={updatePosition} draggable={true}>
-        <AvatarDOM
-          id={selfUserInfo.id}
-          key={0}
-          multiplier={selfUserInfo.multiplier}
-          _backgroundColor={selfUserInfo.avatarColor.background}
-          _borderColor={selfUserInfo.avatarColor.border}
-          isSelf={true}
-          initial={selfUserInfo.name[0]}
-          active={selfUserInfo.active}
-          mute={selfUserInfo.mute}
-        />
-      </Draggable>
-      {
-        userInfos.map((info, index) => {
-          if (!info) {
-            info = defaultUserInfo;
-          }
-          return (
-            <Draggable position={{ x: info.position[0], y: info.position[1] }} onPositionChange={null} draggable={false} key={index}>
-              <AvatarDOM
-                id={info.id}
-                key={index + 1}
-                multiplier={info.multiplier}
-                _backgroundColor={info.avatarColor.background}
-                _borderColor={info.avatarColor.border}
-                isSelf={false}
-                initial={info.name[0]}
-                active={info.active}
-                mute={info.mute}
-              />
-            </Draggable>
-          );
-        })
-      }
+      {userInfos.map((info, index) => {
+        if (!info) {
```

Comment:
We can make a follow up ticket for this, but probably should just delete the user if the info doesn't exist, instead of creating a grey (or whatever the default was), empty avatar.

Reply:
Updated with https://github.com/Amijakan/mochatto/pull/296/commits/631c08f31d3fa785e6f3b2089b43e894ec46b923